linode/instance/resource.go

```diff
@@ -821,6 +830,11 @@ func updateResource(ctx context.Context, d *schema.ResourceData, meta interface{
 		}
 	}
 
+	if d.HasChange("ipv4") {
+		// Log a warning that direct IP changes are not supported
+		return diag.Errorf("Changing IPv4 addresses after instance creation is not directly supported. The attempted change will be ignored.")
+	}
+
```

Comment:
Would it be better to use [`ForceNew`](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#forcenew) attribute to let TF recreated the resource when the field is updated?

Reply:
Given that the update function is designed to modify existing resources, when a customer attempts to update a resource by adding an IPv4 field, it's more likely they intend to assign a reserved IPv4 address to the existing resource rather than create a new Linode instance with a reserved IP address. However, since this specific operation is not currently supported, we've decided to address this by displaying a warning message and preventing the operation from proceeding. This approach ensures that customers are informed about the limitation

Reply:
I agree. I will make the ipv4 attribute a ForceNew one so that a new linode will be created any time the ipv4 field is updated.